pkgs/by-name/gi/git-blame-someone-else/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+}:
+
+stdenv.mkDerivation {
+  pname = "git-blame-someone-else";
+  version = "0";
+
+  src = fetchurl {
+    url = "https://github.com/jayphelps/git-blame-someone-else/archive/refs/heads/master.zip";
+    hash = "sha256-jZkUKv2SV28wsM18tCqNxoCZmLxdYH2Idh9RLibH2yA=";
+  };
+
+  doCheck = true;
+
+  doInstallCheck = true;
+
+  meta = {
+    description = "Blame someone else for your bad code.";
+    longDescription = ''
+      A simple script to spoof a git commit as someone else. Created as a joke, not recommended for production use.
+    '';
+    homepage = "https://github.com/jayphelps/git-blame-someone-else";
+    license = lib.licenses.mit;
+    maintainers = [ "https://github.com/WGrav01" ];
+    mainProgram = with lib.maintainers; [ wgrav01 ];
```

Comment:
`mainProgram` seems to be invalid.

Reply:
Yep, typo. Meant to put that in maintainers